apps/modernization-ui/src/apps/page-builder/components/ValuesetLibrary/ValuesetLibrary.tsx

```diff
@@ -0,0 +1,43 @@
+import { PageProvider } from 'page';
+import React, { useState } from 'react';
+import './ValuesetLibrary.scss';
+import './ValuesetTabs.scss';
+import { ValuesetLibraryTableWrapper } from './ValuesetLibraryTableWrapper';
+export const ValuesetLibrary = () => {
+    const [pageSize] = useState(10);
+    const [activeTab, setActiveTab] = useState('local');
+    const handleTab = (tab: string) => {
+        setActiveTab(tab);
+    };
+
+    return (
+        <>
+            <div className="valueset-local-library">
+                <div className="margin-left-2em">
+                    <h2>Add value set</h2>
+                </div>
+                <ul className="tabs">
+                    <li className={activeTab == 'local' ? 'active' : ''} onClick={() => handleTab('local')}>
+                        Search Local
+                    </li>
+                    <li className={activeTab == 'phin' ? 'active' : ''} onClick={() => handleTab('phin')}>
+                        Search PHIN VADS
+                    </li>
+                    <li className={activeTab == 'recent' ? 'active' : ''} onClick={() => handleTab('recent')}>
+                        Recently Created
```

Comment:
The designs called for multiple tabs but there has been some discussion on merging the local and recently created tab and adding some filters to the page.


For the PHIN VADS tab, the idea is to utilize the PHIN VADS API you created, not the standard find value sets API. --- However, the PHIN VADS API has been put on the back burner until everything else is finished up. So I'm going to recommend removing the PHIN VADS tab.

Reply:
Thanks Mike. I will fix and commit the changes